pom.xml

```diff
@@ -228,8 +228,7 @@
     <version.org.jboss.xnio>3.7.7.Final</version.org.jboss.xnio>
     <version.org.jdom>1.1.3</version.org.jdom>
     <version.org.jfree.jfreechart>1.5.0</version.org.jfree.jfreechart>
-    <version.org.jpmml.model>1.4.11
-    </version.org.jpmml.model> <!-- jpmml-model BSD 3C license - ATTENTION 1.4.11 intentional, because 1.4.9 evaluators depend on 1.4.11 -->
+    <version.org.jpmml.model>1.5.1</version.org.jpmml.model> <!-- jpmml-model BSD 3C license - ATTENTION 1.4.11 intentional, because 1.4.9 evaluators depend on 1.4.11 -->
```

Comment:
Did you want to update the comment too?

Reply:
.. do you think `because 1.4.9 evaluators depend on 1.4.11` is important?

Reply:
@manstis 
Yes - even if I do not understand the meaning of it.
I think they should be the same version.
Does anybody of you have any clue why they have different versions ?
@tarilabs ^ ^

Reply:
answered here: https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1374#issuecomment-648843512